src/main/java/edu/kis/powp/jobs2d/drivers/DriverManager.java

```diff
@@ -30,4 +33,17 @@ public synchronized void setCurrentDriver(Job2dDriver driver) {
 	public synchronized Job2dDriver getCurrentDriver() {
 		return currentDriver;
 	}
```

Comment:
Why do you allow to use current driver and composite driver separately? I think it is very hard for clients to now how to use driver manager properly